src/main/kotlin/org/taktik/icure/services/external/rest/v1/controllers/support/InsuranceController.kt

```diff
@@ -65,6 +65,13 @@ class InsuranceController(
 			?: throw ResponseStatusException(HttpStatus.INTERNAL_SERVER_ERROR, "Insurance deletion failed")
 	}
 
+	@Operation(summary = "Gets all insurances")
+	@GetMapping
+	fun getInsurances(): Flux<InsuranceDto> {
```

Comment:
No, this service should never exist : Returning all the insurances entities of the database completely bypass any check if current HCP may access them or not, and could encounter performance issues. 

Best way to list insurances would be to create a 
`POST /filter` service API, with the following filters : 
- InsuranceByHcPartyFilter;
- InsuranceByIdsFilter (optional, same as `/byIds`service API)